### PR TITLE
Inboxsdk types

### PR DESCRIPTION
**Pull request by amiram**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.inboxsdk.com/
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

#

### CODE REVIEW

**Comment by amiram** [comment]
#fix #11357

File: types/inboxsdk/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,1072 @@
+// Type definitions for InboxSDK 2.0
+// Project: https://www.inboxsdk.com/
+// Definitions by: Raphaël Doursenaud <https://github.com/rdoursenaud>
+//                 Amiram Korach <https://github.com/amiram>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/*
+ * Copyright (c) 2016 GPC.solutions
```
Comment (rdoursenaud) [inline types/inboxsdk/index.d.ts]: @amiram I believe you should add your own copyright ;)

Comment (amiram) [reply types/inboxsdk/index.d.ts]: Sure. Do you want me to put your name as a maintainer? You'll get an email for approving each pull request to this code.

Comment (rdoursenaud) [reply types/inboxsdk/index.d.ts]: I don't think I'm invested in InboxSDK enough but if you believe I can help, why not.

**Pull request merged**